controllers/bsl.go

```diff
@@ -211,8 +211,12 @@ func (r *DPAReconciler) UpdateCredentialsSecretLabels(secretName string, namespa
 		secret.Labels[oadpv1alpha1.OadpOperatorLabel] = "True"
 		needPatch = true
 	}
-	if secret.Labels[namespace+".dataprotectionapplication"] != dpaName {
-		secret.Labels[namespace+".dataprotectionapplication"] = dpaName
+	if secret.Labels["dataprotectionapplication.name"] != dpaName {
+		secret.Labels["dataprotectionapplication.name"] = dpaName
+		needPatch = true
+	}
+	if secret.Labels["dataprotectionapplication.namespace"] != namespace {
```

Comment:
do we need to store this info in the secret :question: is not the same namespace it lives in :question: